### ISSUE TITLE
Visual artifacts around JInternalFrames

### ISSUE THREAD

**Issue by kirill-grouchnikov** [JFormDesigner/FlatLaf#11]
<img width="454" alt="image" src="https://user-images.githubusercontent.com/52874/67026616-266f3280-f0d6-11e9-8881-4c61f668cc1e.png">
<img width="489" alt="image" src="https://user-images.githubusercontent.com/52874/67026625-2c651380-f0d6-11e9-97c0-3f4501c97f14.png">

**Comment by DevCharly** [comment]
Thanks, I'll have a look. 

`JInternalFrame` support is not yet implemented.

**Comment by rogerbj** [comment]
Thanks for making FlatLaf available, really impressive! Are there any plans for internal frame support any time soon?

**Comment by DevCharly** [comment]
Thanks, Roger.

Internal frame support is coming soon, but I'm not sure whether this will happen before Christmas.
Worked already some hours on it, but this takes much more time...

**Comment by rogerbj** [comment]
I am excited to hear that!

**Comment by Chrriis** [comment]
I, too, am very interested in internal frame support. It is the only missing feature before we can ditch our current L&F and replace it with FlatLaf. I can assist in testing as soon as there is something to play with.

**Comment by Chrriis** [comment]
This original bug report was about visual artifacts around Internal frames, and this seems fixed.

I wonder if a different bug report needs to be opened about internal frame decorations support.
Here is a simple test case:
```Java
    public static void main(String[] args) throws Exception {
        UIManager.setLookAndFeel(new FlatLightLaf());
        JFrame frame = new JFrame();
        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        JDesktopPane desktopPane = new JDesktopPane();
        JInternalFrame internalFrame = new JInternalFrame("Test", true, true, true, true);
        internalFrame.setLocation(50, 30);
        internalFrame.setSize(300, 200);
        desktopPane.add(internalFrame);
        frame.getContentPane().add(desktopPane);
        frame.setSize(400, 300);
        frame.setVisible(true);
        internalFrame.setVisible(true);
    }
```

And here is the result:
![InternalFrame](https://user-images.githubusercontent.com/1174202/72105333-aa2f6880-332d-11ea-9072-684cf40f8ba4.png)

As you can see, the internal frame decorations are not FlatLaf.

**Comment by DevCharly** [comment]
> I wonder if a different bug report needs to be opened about internal frame decorations support.

Yes, would you like to open one?

**Comment by DevCharly** [comment]
fixed in 0.27
